src/overlays/actors/ovl_En_Firefly/z_en_firefly.c

```diff
@@ -769,7 +769,7 @@ void EnFirefly_PostLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3s*
     EnFirefly* this = THIS;
 
     if ((this->currentType != KEESE_FIRE) && (limbIndex == 27)) {
```

Comment:
```suggestion
    if ((this->currentType != KEESE_FIRE) && (limbIndex == FIRE_KEESE_LIMB_HEAD)) {
```
Could you also replace the other magic numbers, with the enum values? Should be anything that `limbIndex` gets compared to.